tests/rptest/tests/metrics_reporter_test.py

```diff
@@ -141,7 +141,7 @@ def assert_fields_are_the_same(metadata, field):
         assert last['active_logical_version'] == features['cluster_version']
         assert last['original_logical_version'] == features[
             'original_cluster_version']
-        assert last['has_valid_license'] == False
+        assert last['has_valid_license']
```

Comment:
i don't know specifically, but it just seems like if that conditional was fixed and this was the only test that needed updated, that maybe there should be more test cases? maybe not tho, just a hunch

Reply:
worth noting that this change is isolated to phone-home telemetry. so I'm not sure there's anything else to test in terms of RP behavior. i'll give it some thought though

Reply:
ahh i see. that makes sense